### ISSUE TITLE
Log message: No endpoints ruleset found for service s3, falling back to legacy endpoint routing.

### ISSUE THREAD

**Issue by zichd** [IBM/ibm-cos-sdk-python#56]
```
import logging
logging.getLogger().setLevel(logging.INFO)
import ibm_boto3
from ibm_botocore.client import Config

cos_instance = ibm_boto3.resource(
    service_name="s3",
    ibm_api_key_id="*********",
    ibm_service_instance_id="***********",
    ibm_auth_endpoint="*********",
    config=Config(signature_version="oauth"),
    endpoint_url="********"
)

```

This code renders this log message with ibm-cos-sdk==2.13.1:
`INFO:ibm_botocore.client:No endpoints ruleset found for service s3, falling back to legacy endpoint routing.`

I didn't find any functional impact other than the message itself. If I switch back to 2.13.0, the message disappears. Should I update my code somehow or is it a bug?

**Comment by arnabm28** [comment]
Hello,

Thanks for pointing out the issue. We have seen this additional log message being printed after updating to the latest version of the python core package (2.13.1).

We are working on this issue internally and will come up with more updates.

However additionally our observation is, this log is only printed when using ibm_boto3.resources. With ibm_boto3.client the log will not be printed.(This is just a quick observation).

We will update with further details at the earliest.

**Comment by arnabm28** [comment]
Hello,

We have identified the root cause of the issue and we have fixed the same. However owing to the nature of the issue and its current impact, we will have this fix go in our next release cycle.

We will update you once this the new release is out.

Thanks

**Comment by IBMalok** [comment]
@zichd 
We fixed the issue in version 2.13.2. Please verify once and close the ticket. Thanks

**Comment by IBMalok** [comment]
Closing as the issue has been fixed.
